src/components/icons/CheckSquare.tsx

```diff
@@ -0,0 +1,47 @@
+import { IconoirContext } from 'iconoir-react';
+import React from 'react';
+
+function SvgCheckSquare(
+    passedProps: React.SVGProps<SVGSVGElement>,
+    svgRef?: React.Ref<SVGSVGElement>
+) {
+    const context = React.useContext(IconoirContext);
+    const props = {
+        ...context,
+        ...passedProps,
+    };
+
+    return (
+        <svg
```

Comment:
I'm just curious about this. Why is this inlined versus being a separate `check-square.svg` resource? Would we ever need to edit this?